hbase-procedure/src/main/java/org/apache/hadoop/hbase/procedure2/ProcedureExecutor.java

```diff
@@ -1577,15 +1677,6 @@ private LockState executeRollback(Procedure<TEnvironment> proc) {
       LOG.error(HBaseMarkers.FATAL, "CODE-BUG: Uncaught runtime exception for " + proc, e);
     }
 
-    // allows to kill the executor before something is stored to the wal.
-    // useful to test the procedure recovery.
-    if (testing != null && testing.shouldKillBeforeStoreUpdate()) {
```

Comment:
We have replaced `shouldKillBeforeStoreUpdate` here with `shouldKillBeforeStoreUpdateInRollback`?

Reply:
Yes. I split the condition for normal execution and rollback, so in some tests we need to set both, for example, in TestProcedureRecovery.